### ISSUE TITLE
Issue with runtime control time format

### ISSUE THREAD

**Issue by Resinchem** [kalkih/mini-media-player#295]
This issue is inconsistent, but I believe it started with HA 0.108.x:
![image](https://user-images.githubusercontent.com/55962781/80705441-73d05180-8ab4-11ea-9e80-9b6f21e59de5.png)

Sometimes the runtime formats correctly, sometimes it does not.  I have been unable to correlate what causes it to format correctly and what causes it to show in the format listed above.  If it shows correctly, it remains correct until an HA restart.  But when in the state above, it also remains in that state until an HA restart.  Consistent behavior seen across multiple displays/devices (desktop, tablet, mobile).

HA: 0.108.9  Mini-Media Player: v1.7.0

**Comment by kalkih** [comment]
What media player platform are you see this with?

**Comment by Resinchem** [comment]
In this particular case, all the media players are Directv receivers.  I'm still seeing the issue intermittently with HA 0.109.4.  I still haven't been able to determine what causes it to display incorrectly, nor what causes it to correct itself and show the correct run time.  I plan on upgrading the mini-media player to 1.8.1 when I upgrade HA to 0.110 and can report back if the behavior changes with that.  One final note that I noticed just the other day.  If I click and hold on the runtime indicator when it is showing incorrectly, it displays the correctly formatted runtime, until I release the mouse... at which point it reverts back to the incorrect format.

**Comment by kalkih** [comment]
Okay, I don't own any directv devices and have unfortunately not been able to reproduce the issue with my own devices yet.

> One final note that I noticed just the other day. If I click and hold on the runtime indicator when it is showing incorrectly, it displays the correctly formatted runtime, until I release the mouse... at which point it reverts back to the incorrect format.

Interesting 🤔 


Could you check what the device attributes `media_position` and `media_duration` are set to when you see the issue next time? You can find those by navigating to `http://HA-IP:PORT/developer-tools/state`, thanks!

**Comment by Resinchem** [comment]
Will do.  Currently, they are showing perfectly fine.  This really isn't a big deal for me and I still love the player and will keep it regardless.  I simply reported it in case others were also experiencing it. If I'm the only one reporting it, then no problem if you want to close the issue.  Otherwise, I'll post back here with the requested info the next time I see the strange display.

**Comment by Resinchem** [comment]
Well, that didn't take long!  Was doing some general VM maintenance that required an HA restart.  Upon restart, the display was messed up again.  Here what the services show:

media_content_id: 11287049
media_content_type: tvshow
media_duration: 1800
media_position: 158
media_position_updated_at: 2020-05-24T00:25:47.828764
media_title: Diners, Drive-Ins and Dives
media_series_title: Pork, Pasta and Barbecue
media_channel: foodHD (231)
source: 231
media_currently_recording: false
media_rating: TV-G
media_recorded: false
media_start_time: 2020-05-23T20:30:00-04:00
friendly_name: Directv Basement
supported_features: 21425

Display for this receiver currently shows:
![image](https://user-images.githubusercontent.com/55962781/82743121-ceb43c00-9d34-11ea-8346-e724e02d191f.png)

But if I click-hold on the runtime bar with the mouse, the display shows:
![image](https://user-images.githubusercontent.com/55962781/82743152-3a96a480-9d35-11ea-8d07-802b7e00a58b.png)

It reverts to the first image as soon as I release the mouse button.

**Comment by kalkih** [comment]
Is this still an issue?

**Comment by Resinchem** [comment]
I don't believe I've seen the issue for quite some time, but as I said it was inconsistent so I can't say 100% that it doesn't still occur on occasion.  But I think you can close the issue for now.  However, somewhere along the way (I can't say when) the progress bar reverted to always showing 100% progress, regardless of how much of the program has elapsed:
![image](https://user-images.githubusercontent.com/55962781/103844199-5b62fe80-5067-11eb-8d05-d523943dd9de.png)

I actually just noticed this when checking on the prior issue.  I guess I should open as a separate issue... but again, doesn't really affect functionality for my use.  Thanks for a great HA custom card!
